pubspec.yaml

```diff
@@ -11,72 +11,12 @@ environment:
 dependencies:
   collection: '>=1.15.0-nullsafety <1.15.0'
 
-dev_dependencies:	
-  fake_async: ^1.0.0	
-  stack_trace: ^1.0.0	
-  test: ^1.0.0	
-  pedantic: ^1.0.0
+dev_dependencies:
+  fake_async: ^1.1.0-nullsafety
+  stack_trace: ^1.10.0-nullsafety
+  test: ^1.16.0-nullsafety
+  pedantic: ^1.10.0-nullsafety
 
 dependency_overrides:
-  # Dev dep overrides
-  clock:
-    git:
-      url: git://github.com/dart-lang/clock.git
-      ref: null_safety
   fake_async:
```

Comment:
fake_async has also been published so this should be removable

Reply:
Hmm, `fake_async` has both `1.1.0` and `1.1.0-nullsafety` published.

Reply:
https://github.com/dart-lang/fake_async/pull/23